tests/test_main.py

```diff
@@ -379,3 +380,14 @@ def test_logger(tmp_dir, mocker, monkeypatch):
 
     live = Live(resume=True)
     logger.info.assert_called_with("Resumed from step 0")
+
+
+def test_no_log_summary(tmp_dir):
```

Comment:
I am not sure I understand - does this test name mean dvclive should not log `summary.json` but `metrics.json`?

Reply:
The summary file is currently called `metrics.json` (could be changed though).

The test is for the workflow where you can write to the summary file without calling `live.log` but rather editing the `summary` property directly.

Reply:
I guess we could add here a functionality checking that `foo.tsv` does not exist - to make sure that values added via dict modification don't get logged into `tsv` files. WDYT?